qiita_pet/handlers/study_handlers/listing_handlers.py

```diff
@@ -112,11 +121,12 @@ def _build_single_proc_data_info(proc_data_id, data_type, samples):
     dict
         The information for the processed data, in the form {info: value, ...}
     """
-    proc_data = ProcessedData(proc_data_id)
-    proc_info = proc_data.processing_info
+    proc_data = Artifact(proc_data_id)
+    # TODO plugin:
```

Comment:
No, the issue is that some of this changes depends on my changes for the plugin, but the general qiita_pet update should not be blocked due to this. This way, it is easier that we have most of the things changed and I can just search for the TODO and connect the things together in a subsequent PR.